obs-packaging/scripts/obs-pkgs.sh

```diff
@@ -16,3 +15,7 @@ OBS_PKGS_PROJECTS=(
 	ksm-throttler
 	runtime
 )
+
+if [ "$GO_ARCH" == "amd64"]; then
```

Comment:
I'd like to not depend on  `GO_ARCH`, but can we use `uname -m` this will help to any script that source this doe snot depend on an external variable.

Reply:
@jcvenegas : Updated.